Makefile

```diff
@@ -19,6 +19,19 @@ test: generate
 	@go test -v -covermode=count -coverprofile=coverage.out ./pkg/... ./cmd/...
 	@test -z $(COVERALLS_TOKEN) || goveralls -coverprofile=coverage.out -service=circle-ci
 
+.PHONY: lint
+lint:
+	gometalinter --exclude=^vendor\/  ./...
+
+.PHONY: install-lint-deps
+install-lint-deps:
+	curl https://git.io/vp6lP | sh
```

Comment:
Could you please add `github.com/alecthomas/gometalinter` to https://github.com/weaveworks/eksctl/blob/master/build/Gopkg.toml and https://github.com/weaveworks/eksctl/blob/master/build/install.sh instead? That way all the dependencies will be included in the build image, or installed locally via `make install-build-deps`.

Reply:
Seems like and easy thing to do but been struggling getting this working. I must be doing something stupid!

Reply:
happy to pick this up, would be nice to include in 0.1.2!